opaque_keys/edx/keys.py

```diff
@@ -77,6 +77,26 @@ def block_type(self):  # pragma: no cover
         raise NotImplementedError()
 
 
+class CourseKeyV2(OpaqueKey):
+    """
+    An :class:`opaque_keys.OpaqueKey` identifying a
+    serialized Course Key object.
+    """
+    KEY_TYPE = 'course_key_v2'
+    __slots__ = ()
+
+    @abstractproperty
+    def from_course_key(self, course_key):  # pragma: no cover
```

Comment:
Call this `from_course_run_key`. Eventually, course-key-v1 will be replaced with course-run-key-v1.